src/audio/tone.c

```diff
@@ -45,6 +45,7 @@
 
 static const struct comp_driver comp_tone;
 
+
 /* 04e3f894-2c5c-4f2e-8dc1-694eeaab53fa */
```

Comment:
There is a stray new line here :)

Reply:
Use `draft:false` to ignore draft distractions :-)